pytorch_lightning/accelerators/accelerator.py

```diff
@@ -510,6 +510,15 @@ def setup_optimizers_in_pre_dispatch(self) -> bool:
         """
         return self.training_type_plugin.setup_optimizers_in_pre_dispatch
 
+    @property
+    def restore_checkpoint_after_pre_dispatch(self) -> bool:
+        """
+        Override to delay restoring from checkpoint till after predispatch.
+        This is useful when the plugin requires all the setup hooks to run before loading checkpoint.
+        Returns: If true, restore checkpoint after pre_dispatch.
```

Comment:
```suggestion

        Returns: 
            If true, restore checkpoint after pre_dispatch.
```
for correct rendering, this whitespace is needed afaik